apps/website/prisma/schema.prisma

```diff
@@ -301,7 +301,7 @@ model ShortLinksTracking {
 }
 
 model CalendarEvent {
-  id          String @id @default(cuid())
+  id          String   @id @default(cuid())
```

Comment:
I guess the IDE wanted this indented. Can we keep it like this?

Reply:
Yeah, seems right, its adding the extra space because of the `?` on one of the types further down in this model

Reply:
Its aligned with `DateTime` even further down. Is it worth running `pnpm prisma format` in lint staged?

Reply:
Yeah sounds worth adding to lint-staged and the package scripts, in a follow-up pr